src/config/firebase-utils.js

```diff
@@ -3,21 +3,26 @@ import { auth, db } from './firebase-config';
 import { createUserWithEmailAndPassword, updateProfile, deleteUser, signInWithEmailAndPassword } from 'firebase/auth';
 import { toast } from 'react-toastify';
 
-export const signupWithFirebase = async ({ email, password, displayName, photoURL }) => {
+export const signupWithFirebase = async ({ email, password }) => {
   try {
     const userCredential = await createUserWithEmailAndPassword(auth, email, password);
     const user = userCredential.user;
-    await updateProfile(user, {
-      displayName,
-      photoURL,
-    });
     return user;
   } catch (e) {
     console.error('Error in signupWithFirebase: ', e);
     toast.error(e.message);
   }
 };
 
+export const updateUser = async(user, data) => {
+  try {
+    await updateProfile(user, data);
+  } catch (error) {
+    toast.error("Failed to update user")
+    console.error(`Failed to update user: ${error}`);
+  }
```

Comment:
Why  do we  need prints in production

Reply:
Why not? 
It explains the reason for the failure in console but and only alert the user (without the details)